spec/20-http_request_header_format.md

```diff
@@ -176,6 +174,28 @@ There are two additional options that vendors MAY follow:
 - A component that makes a deferred or delayed recording decision may communicate the priority of a recording by setting `sampled` flag to `1` for a subset of requests.
 - A component may also fall back to probability sampling and set the `sampled` flag to `1` for the subset of requests.
 
+##### Random Trace ID Flag
+
+<!--
+
+TODO: how many random bytes are needed?
+7 was chosen as it can be efficiently represented as a 64-bit signed or unsigned integer.
+8 would require an unsigned long which is not supported by some languages (like Java).
+63 bits would be possible, but would require a more complex description that may be more difficult to understand.
```

Comment:
63 was not only because of java. Also golang has this limitation https://pkg.go.dev/math/rand#Int63

Reply:
It's ok. This is just a comment to help us make a decision. Once we make a decision we can document the reasons.